src/olympia/migrations/929-add-user-auth-id.sql

```diff
@@ -0,0 +1,3 @@
+-- note: the column is added as nullable, a command will backfill values for
+-- existing users.
+ALTER TABLE `users` ADD COLUMN `auth_id` integer UNSIGNED;
```

Comment:
nitpick but given that there's a potential maximum and especially since it's "just" a regular random value I wonder if this should be a `UNIQUE` value too. If by some coincidence two users end up with the same auth_id and tokens use them I guess the tokens security is somewhat compromised too or at least it's weaker.